### DISCUSSION TITLE
Charging: 3 minutes reported

### DISCUSSION THREAD

**Discussion by jolll-dk** [teslamate-org/teslamate#3635] [Q&A] [answered]
Hi :)

Since the update to firmware 2023.44.30.9 6 days ago,
my Teslamate only report the last 3 minutes of a charging session.
Charging is reset every 3 minutes in Teslamate, and the counter starts again from zero seconds.
Anyone else seen this issue?

Regards Bryan

**Comment by commanderi1972** [comment]
I have exactly the same problem. But only when I charge at home.![image](https://github.com/teslamate-org/teslamate/assets/157980603/feaba513-41a3-4163-9310-4d88fbc9d660)

**Reply by jolll-dk** [reply]
On my setup it's both at home and at superchargers :(

**Reply by commanderi1972** [reply]
very strange, seems to be a bug. Let's see if the developer can say something about it.

**Reply by jolll-dk** [reply]
Hmm, I see it's the same under Drives 
No trip is longer than 3-4 minutes.

**Reply by jolll-dk** [reply]
This image seems to fix the problem :)

version: "3"

services:
  teslamate:
    image: ghcr.io/teslamate-org/teslamate:pr-3630

**Comment by cwanja** [comment]
What do your TeslaMate container logs report? 

Beginning to wonder if this is the first firmware that rate limits API calls.

**Comment by wooter** [comment, marked as answer]
Related to a change in the API endpoint, see https://github.com/teslamate-org/teslamate/issues/3629. A fix is in the works, https://github.com/teslamate-org/teslamate/pull/3630.
